internal/server/server.go

```diff
@@ -442,7 +442,7 @@ func getBoolParam(url *url.URL, name string, ifEmpty bool) bool {
 	}
 
 	for _, x := range p {
-		if strings.ToLower(x) == "true" {
+		if strings.EqualFold(x, "true") {
```

Comment:
TIL about [`strings.EqualFold`](https://cs.opensource.google/go/go/+/go1.25.1:src/strings/strings.go;l=1194), which does case-insensitive string compare for unicode! :slightly_smiling_face: